shared-libs/transitions/src/transitions/utils.js

```diff
@@ -58,7 +58,12 @@ module.exports = {
   },
   isIdUnique: (id) => {
     return db.medic
-      .query('medic-client/contacts_by_reference', { key: [ 'shortcode', id ]})
+      .query('medic-client/contacts_by_reference', { key: ['shortcode', id] })
+      .then(results => !(results && results.rows && results.rows.length));
+  },
+  isPhoneUnique: (phoneNumber) => {
```

Comment:
It looks like the `transitions/utils` file already imports '@medic/validation' so if you expose the new `uniquePhone` function there we can reuse this code.

Reply:
@garethbowen The code in utils was redundant hence I removed it. Could have been done by putting the duplicate check in utils but couldn't find any way to stop all further transitions when phone was duplicate.